helm-charts/basehub/values.yaml

```diff
@@ -713,67 +716,63 @@ jupyterhub:
 
         async def profile_list_allowed_teams_filter(spawner):
             """
-            Returns the initially configured profile_list filtered based on if
-            the spawning user is part the profiles' specified GitHub org/teams.
-
-            Adds a 'allowed_teams' key to profile_list, with a list of GitHub teams (of the form
-            org-name:team-name) for which the profile is made available.
-
-            If the user isn't part of any team whose membership grants them access to even a single
-            profile, they aren't allowed to start any servers.
+            Returns the initially configured profile_list filtered based on the
+            user's membership profiles' `allowed_teams` for profiles its
+            configured. `allowed_teams` is a list of GitHub organizations and/or
```

Comment:
This sentence doesn't make sense

Reply:
I think my brain caught up to what a profile was about halfway through reviewing the PR. What I think this docstring is trying to say is roughly:

Return the initially configured list of profiles filtered by whether or not a user is a member of a team defined in `allowed_teams`, provided `allowed_teams` has been configured for a given profile.

Reply:
Yes, thats it! I've already pushed something somewhat more readable. I'll go for a merge now and reach out to smithsonian community champs next.